app/views/consent_forms/confirm.html.erb

```diff
@@ -152,6 +154,36 @@
             href: session_consent_form_edit_path(@session, @consent_form, "contact-method"),
             visually_hidden_text: "your phone contact method")
         end
+      else
+        summary_list.with_row do |row|
+          row.with_key { "Phone number" }
+          row.with_value { "Not provided" }
+          row.with_action(text: "Change",
+            href: session_consent_form_edit_path(@session, @consent_form, :parent),
+            visually_hidden_text: "your phone")
+        end
+      end
```

Comment:
This is unrelated to this PR, just aligning to the prototype in another small way.